pootle/core/initdb.py

```diff
@@ -45,13 +48,13 @@ def _init_db(self, create_projects=True):
         self.create_essential_users()
         self.create_root_directories()
         self.create_template_languages()
+        self.create_default_languages()
         if create_projects:
             self.create_terminology_project()
         self.create_pootle_permissions()
         self.create_pootle_permission_sets()
         if create_projects:
             self.create_default_projects()
-        self.create_default_languages()
```

Comment:
Moving this up looks like fixing a bug. Maybe put in a separate commit is better.